src/main.js

```diff
@@ -230,12 +207,45 @@ function switchScreens(mainView, alternateView) {
   alternateView.classList.toggle('hidden')
 }
 
-// function posterFormToggle() {
-//   document.querySelector('.main-poster').classList.toggle('hidden')
-//   document.querySelector('.poster-form').classList.toggle('hidden')
-// }
+// created new function for adding event listeners to each mini poster and
+// for deleting mini posters
+function deleteMiniPoster() {
+  var miniPosters = document.querySelectorAll('.mini-poster')
+
+  for (var i = 0; i < miniPosters.length; i++) {
+    miniPosters[i].addEventListener('dblclick', function() {
+      var thisPosterID
+
+      if (event.target.id){
+        thisPosterID = event.target.id
+      } else {
+        thisPosterID = event.target.parentElement.id
+      }
 
-// function savedPostersToggle() {
-//   document.querySelector('.main-poster').classList.toggle('hidden')
-//   document.querySelector('.saved-posters').classList.toggle('hidden')
-// }
+      for (var i = 0; i < savedPosters.length; i++) {
+        if (savedPosters[i].id == thisPosterID){
```

Comment:
We actually were wanting the == because the savedPosters[i].id is a string and thisPosterID is an integer. Usually though we tried to use === where possible!